packages/core/stories/avatar/avatar.doc.stories.mdx

```diff
@@ -102,4 +102,10 @@ The avatar component can contain an `[Icon]`(?path=/docs/documentation-icons-ico
 
 <CharacteristicUsage of={Avatar} />
 
+## Accessibility
+
+Decorative avatars (avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an avatar for decorative purposes, pass `aria-hidden=”true”` to hide its label from the screen reader.
+
+If an alternative icon is used inside an avatar, the icon's `aria-label` should be descriptive in relation to the avatar context, not the icon context. For example, when using `<BuildingSolidIcon/>` to represent a company `aria-label` should read "company profile" instead of the icon default label "building solid".
```

Comment:
@rufcha can I please ask for a review in this new paragraph for avatar's documentation?